core/src/test/java/io/hyperfoil/core/generators/RandomStringStepTest.java

```diff
@@ -0,0 +1,41 @@
+package io.hyperfoil.core.generators;
+
+import static org.junit.Assert.assertNotNull;
+
+import java.util.List;
+
+import org.junit.Test;
+
+import io.hyperfoil.api.config.Locator;
+import io.hyperfoil.api.config.Step;
+import io.hyperfoil.api.session.Access;
+import io.hyperfoil.api.session.Session;
+import io.hyperfoil.core.session.SessionFactory;
+import io.hyperfoil.core.test.TestUtil;
+
+public class RandomStringStepTest {
+
+   @Test
+   public void testStringGenerator() {
+      // builder
+      String[] vars = new String[]{ "first", "second", "third", "fourth" };
```

Comment:
Does using 4 different var names test anything extra to using just single variable?
Btw. `var` is not the best name for a variable since it's a keyword (though positional to not break backwards compatibility) - it's confusing GitHub UI.

Reply:
Test nothing extra but I liked the example with 4 diff names.
Renamed `var` to `varName`